components/facebook_groups/facebook_groups.app.mjs

```diff
@@ -27,16 +30,21 @@ export default {
       label: "Post",
       description: "Identifier of a post",
       async options({ groupId }) {
-        const opts = {
-          groupId,
-        };
-        const { data } = await this.listPosts(opts);
-        return data?.map(({
-          id, message,
-        }) => ({
-          label: message || id,
-          value: id,
-        })) || [];
+        try {
+          const opts = {
+            groupId,
+          };
+          const { data } = await this.listPosts(opts);
+          return data?.map(({
+            id, message,
+          }) => ({
+            label: message || id,
+            value: id,
+          })) || [];
+        } catch (error) {
+          console.error(error);
+          throw new ConfigurationError("Please doublecheck that the Pipedream app has been installed in the group you have selected above. Learn more [here](https://pipedream.com/apps/facebook-groups).");
```

Comment:
```suggestion
          throw new ConfigurationError("Please double-check that the Pipedream app has been installed in the group you have selected above. Learn more [here](https://pipedream.com/apps/facebook-groups).");
```

Reply:
Changed 'doublecheck' to 'double-check' to follow Google's Developer docs style guide (hyphenation).